winui/Cartesian-Charts/FastCharts/Fast-Scatter.md

```diff
@@ -11,7 +11,7 @@ documentation: ug
 
 A fast scatter chart is a special kind of series that can render a collection with huge number of data points using `WriteableBitmap`. [FastScatterBitmapSeries](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.FastScatterBitmapSeries.html) is used to render high number of scatter points. 
```

Comment:
A fast scatter chart is a special kind of series that can render a collection with huge number of data points using the `WriteableBitmap.` The [FastScatterBitmapSeries](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.FastScatterBitmapSeries.html) is used to render high number of scatter points.